omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml

```diff
@@ -0,0 +1,1732 @@
+                                    </excludeIf>
+                                </td>
+                                <td></td>
+                            </tr>
+                            </tbody>
+                        </table>
+                    </section>
+                    <section>
+                        <h4>ART Transfer In</h4>
+                        <table class="multi">
+                            <tbody>
+                            <repeat>
+                                <template>
+                                    <obsgroup groupingConceptId="99065">
+                                        <tr>
+                                            <td class="ARTStartEnableDisable2">
+                                                <obs id="99160" conceptId="99160" labelText="Date transfered:"/>
+                                            </td>
+                                            <td class="ARTStartEnableDisableChild2">
+                                                <obs conceptId="90206" size="20" labelText="Transfered from:"/>
+                                            </td>
+                                            <td colspan="2" class="ARTStartEnableDisableChild2">
+                                <span class="ArtCareEnableDisable1">
+                                <small>
+                                    <obs labelText="ARVs" conceptId="99064"
+                                         answerConceptIds="99015,99016,99005,99006,164979,99039,99040,99041,99042,164977,99007,99008,99044,99043,99282,99283,99046,99017,99018,99019,99045,99284,99285,99286,99046,164978,163017,90002"
+                                         answerLabels="d4T-3TC-NVP,d4T-3TC-EFV,AZT-3TC-NVP,AZT-3TC-EFV,AZT-3TC-DTG,TDF-3TC-NVP,TDF-3TC-EFV,TDF-FTC-NVP,TDF-FTC-EFV,TDF-3TC-DTG,ABC-ddI(250)-LPV/r,ABC-ddI(400)-LPV/r,TDF-3TC-LPV/r,TDF-FTC-LPV/r,ZDV-ddI(250)-LPV/r,ZDV-ddI(400)-LPV/r,ZDV-3TC-LPV/r,ABC-ddI-LPV/r,ABC-ddI-NFV,ABC-ddI-SQV/r,ZDV-ddI-LPV/r,ZDV-ABC-LPV/r,ABC-ddI-ATV/r,AZT-3TC-ATV/r,AZT-3TC-LPV/r,ABC-3TC-DTG,ABC-3TC-LPV/r,Other"/>
+                                </small>
+                                    </span>
+                                                <span class="ArtCareEnableDisableChild1">
+                                    <obs conceptId="99269" size="12" labelText="Other:"/>
+                                </span>
+
+                                            </td>
+                                            <td></td>
+                                            <td align="center">
+                                                <a href="#" class="add">Add</a> | <a href="#" class="remove">Remove</a>
+                                            </td>
+                                        </tr>
+                                    </obsgroup>
+                                </template>
+                                <render/>
```

Comment:
@dbaluku Why the duplicated renders?

Reply:
That is the implementation used even in  the TB form